src/services/auth.js

```diff
@@ -1,6 +1,10 @@
 import * as firebase from 'firebase/app';
 import 'firebase/auth';
 
+import SIGNUP_ENDPOINT from '../constants/endpoints';
+
+const SIGN_UP_URL = `http://localhost:3001${SIGNUP_ENDPOINT}`;

```

Comment:
can you add a TODO to fetch base url based on env instead

Reply:
I'll add that and then push it to newSignupPage then?